### PR TITLE
React/week2

### PR DESCRIPTION
**Pull request by panivita**

### CODE REVIEW

File: react/week1/todo-app/src/components/timer.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, { useState, useEffect } from "react";
+
+export const Timer = () => {
+  const [seconds, setSeconds] = useState(0);
```
Comment (r0hitsharma) [inline react/week1/todo-app/src/components/timer.js:4]: I see you've used hooks here, did you understand the concept?

File: react/week1/todo-app/src/components/todo-section.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  ));
+  return <ul>{todoItems}</ul>;
+};
+
+export const TodoListSection = ({ list = [] }) => {
+  const [todos, setTodos] = useState(list);
+  const addTodo = () => {
+    setTodos((t) => {
+      const item = {
+        id: t[t.length - 1]?.id + 1 || 1,
```
Comment (r0hitsharma) [inline react/week1/todo-app/src/components/todo-section.js:27]: Good use of the operator :+1: But if you're in doubt, feel free to ask.

File: react/week1/todo-app/src/components/timer.js (hunk 1 above)
Comment (panivita) [reply react/week1/todo-app/src/components/timer.js:4]: yes, I understand.

File: react/week1/todo-app/src/components/todo-section.js (hunk 2 above)
Comment (panivita) [reply react/week1/todo-app/src/components/todo-section.js:27]: Thanks :)

**Pull request merged**